apps/builder/app/builder/features/settings-panel/shared.tsx

```diff
@@ -155,7 +156,8 @@ export const Label = ({
 
 export const useLocalValue = <Type,>(
   savedValue: Type,
-  onSave: (value: Type) => void
+  onSave: (value: Type) => void,
+  { autoSave = true } = {}
```

Comment:
why make it optional?

Reply:
to avoid passing it everywhere since so far I didn't see a need to customize it at all, maybe we should just remove it